eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -140,6 +140,7 @@ extends:
                   Commitish: ${{ parameters.SdkRepoCommit }}
                   WorkingDirectory: $(SdkRepoDirectory)
               SkipCheckoutNone: true
+              PreserveAuthToken: true
```

Comment:
Let's put this under the Token as we don't need it otherwise.

Reply:
Updated.